include/sway/layout.h

```diff
@@ -75,4 +75,7 @@ void swayc_log(log_importance_t verbosity, swayc_t *cont, const char* format, ..
  */
 enum swayc_layouts default_layout(swayc_t *output);
 
+inline bool is_auto_layout(enum swayc_layouts layout) {
```

Comment:
Needs to move out of the header and into a file (no code in headers, just don't make it inline)